docs/REST API/rest-api.md

```diff
@@ -9119,11 +9119,11 @@ Get a list of all software.
 | min_cvss_score | integer | query | _Available in Fleet Premium_. Filters to include only software with vulnerabilities that have a CVSS version 3.x base score higher than the specified value.   |
 | max_cvss_score | integer | query | _Available in Fleet Premium_. Filters to only include software with vulnerabilities that have a CVSS version 3.x base score lower than what's specified.   |
 | exploit | boolean | query | _Available in Fleet Premium_. If `true`, filters to only include software with vulnerabilities that have been actively exploited in the wild (`cisa_known_exploit: true`). Default is `false`.  |
-| platform | string | query | Filter software by platform. Supported values are `darwin`, `windows`, and `linux`.  |
+| platform | string | query | Filter software titles by platforms. Options are: `"macos"`, `"windows"`, `"linux"`, `"chrome"`, `"ios"`, `"ipados"`.  |
 
 #### Example
 
-`GET /api/v1/fleet/software/titles?team_id=3`
+`GET /api/v1/fleet/software/titles?team_id=3&platform=darwin&platform=windows`
```

Comment:
We shouldn't repeat the same parameter more than once; if we plan on supporting multiple platform filters in one API request, we can accept a comma-separated string:
```suggestion
`GET /api/v1/fleet/software/titles?team_id=3&platform=darwin,windows`
```

(We should include that info in the parameter description, too.)

Reply:
ah, just saw the comments about this on the other PR. If multiples of the same parameter is more common in Go, I'm cool with that! We should add a note to the parameter description to clarify that you can use the parameter more than once